### PR TITLE
Updating our version to use semver1.0.0 scheme.

### PR DESCRIPTION
**Pull request by pradipd**
This change is [Reviewable](https://reviewable.io/reviews/microsoft/winobjc/1426)

### CODE REVIEW

**Comment by pradipd** [comment]
@bbowman is added to the review. #Closed

File: build/nuget/build.ps1 [hunk 1]
```diff
@@ -21,7 +21,7 @@ if(!$Repo) {
 
 $version = "0.0.1"
```
Comment (rajsesh-msft) [inline build/nuget/build.ps1]: This is not new change, but we are currently 0.0.2 according to build\winobjc.version.

Comment (bbowman) [inline build/nuget/build.ps1]: That is a different version number that I don't really understand. It is what is baked into vsimporter and xib2nib only at the moment. It is independent of the nuget version but we could later bring them in line. Remember that the package @pradipd  is working on here is strictly prototype for now. We will start over the versioning with the new packages I am working on.

Comment (pradipd) [inline build/nuget/build.ps1]: Like Brian mentioned.  this is prototyping.  
We are testing out the workflow using PM's as guinea pigs.

---
In reply to: [88757560](https://github.com/Microsoft/WinObjC/pull/1426#discussion_r88757560) [](ancestors = 88757560)

**Pull request merged**